tests/laplace_matrix_free.hpp

```diff
@@ -257,7 +263,7 @@ void LaplaceMatrixFree<dim, fe_degree, ScalarType>::setup_system(
   }
 
   if (ptree.get("distort_random", false))
-    dealii::GridTools::distort_random(0.2, _triangulation);
+    dealii::GridTools::distort_random(0.1, _triangulation);
```

Comment:
Any reason for changing this?

Reply:
Oh yeah this was a bug. We were using `0.1` for the matrix-based so I just changed to the same value for the matrix-free.